### ISSUE TITLE
interfacing new lie-symmetry paper software

### ISSUE THREAD

**Issue by dlill** [dkaschek/dMod#55]
Would be interesting to interface the software that Clemens sent around today: https://www.mdpi.com/2073-8994/12/3/469
their code is here
https://github.com/GemmaMasFes6/Lie-Symmetries

**Comment by dkaschek** [comment]
Did you check if it extends the possibilities of Bennies current python implementation?

**Comment by dlill** [comment]
No, but Jens said so. Might be something for Severin if he is interested. He's probably the one in the Timmer lab with the most knowledge about Lie groups ...

**Comment by marcusrosenblatt** [comment]
That would be very cool indeed. Best would be to directly link to the new PEtab/SBML-standard and suggest the transformations appropriately. I am planning to do this for the steady state tool as well. Is Bennis tool also on github or is there just the *final* version as we have it in dMod. I will send a link around to the group again. Since the code is in Matlab, parts of the group might tend to use the functions matlab internally... :-)

**Comment by marcusrosenblatt** [comment]
Talking with Severin on monday. Wanna join?
